rs/src/output/data_type.rs

```diff
@@ -857,8 +900,20 @@ impl From<Arc<DataType>> for Parameter {
     }
 }
 
-impl From<u64> for Parameter {
-    fn from(x: u64) -> Self {
-        Parameter::Unsigned(x)
+impl From<i64> for Parameter {
+    fn from(x: i64) -> Self {
+        Parameter::Integer(x)
```

Comment:
I personally am a fan of using `Self` in this position like `Self::Integer(x)`, but that's really just a stylistic preference.

Reply:
I think I personally prefer this method, but I'm honestly not sure if I've been doing it consistently. I think this is just what vscode+Rust analyzer generates, and rustfmt/clippy doesn't complain about it either way. Would be nice if it were opinionated about that. Maybe there's an option I'm not aware of?